llvm/lib/IR/DebugInfoMetadata.cpp

```diff
@@ -1512,7 +1512,6 @@ bool DIExpression::isImplicit() const {
     default:
       break;
     case dwarf::DW_OP_stack_value:
-    case dwarf::DW_OP_LLVM_tag_offset:
```

Comment:
Would this make all expressions with DW_OP_LLVM_tag_offset non implicit?
I don't know implications of this.